models/calendar.yml

```diff
@@ -359,12 +359,16 @@
     - 8:30 PM:
         type: Performance
         emoji: 🎉
-        title: TBD
-        description: TBD
+        title: |
+          open ./guitar.mp4 &
+        description: |
+          Live guitar performance looped through Ableton Live
+          [Find me on Soundcloud](https://soundcloud.com/L1fescape)
         person:
           name: Andrew Kennedy
           twitter: L1fescape
-          pronouns: # examples: she/her, they/them, he/him  (bit.ly/pronoun-faq)
+          github: L1fescape
```

Comment:
IIRC we only support one of {twitter,github} identities right now. Probably should work this requirement into the linter.